addons/nvidia/nvidia.yaml

```diff
@@ -54,4 +51,16 @@ spec:
         initContainers:
         - name: init-wait
           image: busybox
-          command: ['sh', '-c', 'sleep 90']
+          command: ['sh', '-c', 'sleep 180']
```

Comment:
Same comment as https://github.com/mesosphere/kubeaddons-configs/pull/350#discussion_r355059770

Reply:
replied on that thread:)